### PR TITLE
feat: add Moonriver, Metal L2 Testnet

### PR DESCRIPTION
**Pull request by guy93354**
### Description

 Add Moonriver and Metal L2 Testnet using the projects' official docs.

### Backward compatibility

 No

### Testing

 CLI

### CODE REVIEW

File: chains/moonriver/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api-moonriver.moonscan.io/api
+    family: etherscan
+    name: Moonriver Chain Explorer
+    url: https://moonriver.moonscan.io/
```
Comment (paulbalaji) [inline chains/moonriver/metadata.yaml:6]:
```suggestion
    url: https://moonriver.moonscan.io
```

File: chains/metall2testnet/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://testnet.explorer.metall2.com/api
+    family: blockscout
+    name: Metal L2 Testnet Explorer
+    url: https://testnet.explorer.metall2.com/
```
Comment (paulbalaji) [inline chains/metall2testnet/metadata.yaml:6]:
```suggestion
    url: https://testnet.explorer.metall2.com
```

File: chains/metall2testnet/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+domainId: 1740
+isTestnet: true
+name: metall2testnet
+nativeToken:
+  decimals: 18
+  name: Ether
+  symbol: ETH
+protocol: ethereum
+rpcUrls:
+  - http: https://testnet.rpc.metall2.com/
```
Comment (paulbalaji) [inline chains/metall2testnet/metadata.yaml:21]:
```suggestion
  - http: https://testnet.rpc.metall2.com
```

**Comment by guy93354** [comment]
@paulbalaji, sorry Paul, I made a note to remove the final '/', but I forgot... I'll triple-check next time I submit a PR.

**Comment by guy93354** [comment]
@xeno097, can you please check this PR. Paul reviewed it and I corrected the errors.
Thank you.

**Comment by xeno097** [comment]
Hey @guy93354 it seems that the prettier action is failing. Please  run `yarn prettier` and push the changes to make ci pass

**Comment by guy93354** [comment]
Thank you @xeno097, I ran prettier and uploaded the corrected files.

**Pull request merged**